05_data_model_extensions.sql

```diff
@@ -51,6 +35,15 @@ UPDATE qgep_vl.channel_function_hierarchic SET order_fct_hierarchic=12 WHERE cod
 UPDATE qgep_vl.channel_function_hierarchic SET order_fct_hierarchic=11 WHERE code=5074;
 UPDATE qgep_vl.channel_function_hierarchic SET order_fct_hierarchic=14 WHERE code=5075;
 
+-- this column is an extension to the VSA data model and defines whether connected channels are included in inflow/outflow labeling based on function_hierarchic
+ALTER TABLE qgep_vl.channel_function_hierarchic ADD COLUMN include_in_ws_labels boolean DEFAULT FALSE;
```

Comment:
https://github.com/teksi/Home/wiki/TEKSI-Developer-Guide#custom-fields

Reply:
Is a custom field usr_xxx not equal to a db extension by a single user (not included in github and pum)

Reply:
For me it is not totally clear if we have (here)
a) calculated fields -> _ would need an underscore
b) custom field -> usr_xxx 
c) something new - an additional TEKSI needed field, that is not part of VSA-DSS but not realy "calculated" and 
d) do we distinguish if this is done in a view of the model or in the view of the project

Reply:
For me it is c)
A Teksi-specific extension independent of other tables. My suggestion is to prefix these cases x_ (eXtension).